operations/companycontracts.md

```diff
@@ -53,3 +66,20 @@ Returns all contracts between the enterprise and other companies.
 | `IsActive` | boolean | required | Whether the contract is still active. |
 | `CommissionIncluded` | boolean | optional | Whether commission of the travel agency is included in the rate. |
 | `Commission` | number | optional | Commission of the travel agency. |
+| `ChannelManagerAbsoluteAdjustment` | number | optional | Flat fee added to (or subtracted from) the reservation price when coming from Channel Managers. |
+| `ChannelManagerRelativeAdjustment` | number | optional | Percentage of the reservation price added to (or subtracted from) price when coming from Channel Managers. |
+| `Options` | [Travel agency contract options](#travel-agency-contract-options) | required | Options of the travel agency contract. |
+| `AccountingCode` | string | optional | Accounting code of the travel agency contract. |
+| `InvoiceDueInterval` | string | optional | The maximum time, when the invoice has to be be paid in ISO 8601 duration format. |
+| `ChannelManagerBusinessSegmentId` | string | optional | Unique identifier of the [Business segment](businesssegments.md#business-segment).
```

Comment:
@MikeAdamsMews @fpindej @jirrick I'm not 100% sure with this one if ChannelManagerBusinessSegmentId is the business segment or if there are different ones for channel managers.

Reply:
Which means that this description is correct, right?

Reply:
Well it's not wrong, but it doesn't explain it that well. Maybe like `Unique identifier of the Business segment used for incoming reservations coming from Channel managers for this particular contract`? cc @MikeAdamsMews

Reply:
@jirrick your wording sounds good to me, maybe "Unique identifier of the Business Segment used for incoming reservations originating from Channel Managers, for this particular contract."

Reply:
@jirrick @MikeAdamsMews I've used the text form @MikeAdamsMews but added the link to the .md file